semcore/date-picker/src/index.d.ts

```diff
@@ -467,4 +468,127 @@ declare const MonthRangePicker: Intergalactic.Component<
   subtract: (date: number | Date, amount: number, unit: dayjs.OpUnitType) => Date;
 };
 
-export { DatePicker, DateRangePicker, MonthPicker, MonthRangePicker };
+export type DateRangeComparatorProps = DropdownProps &
+  WithI18nEnhanceProps & {
+    /**
+     * The selected date, accepts everything which is accepted by `new Date()`
+     * */
+    value?: DateConstructorParams[];
+    /**
+     * Default value selected date, accepts everything which is accepted by `new Date()`
+     * */
+    defaultValue?: DateConstructorParams[];
+    /**
+     * The compared date, accepts everything which is accepted by `new Date()`
+     * */
+    compare?: DateConstructorParams[];
+    /**
+     * Default value compared date, accepts everything which is accepted by `new Date()`
+     * */
+    defaultCompare?: DateConstructorParams[];
+    /**
+     * Default value date for showing the necessary month
+     * */
+    defaultDisplayedPeriod?: DateConstructorParams;
```

Comment:
looks like period should have two values...

Reply:
Display period is corresponds to date (month or year) that is currently displayed to user. It cannot it be two values because even if two calendars are displayed, it should display sibling dates